src/lib/tooltip/tooltip.ts

```diff
@@ -549,18 +540,16 @@ export class TooltipComponent {
 
   /**
    * Shows the tooltip with an animation originating from the provided origin
-   * @param position Position of the tooltip.
    * @param delay Amount of milliseconds to the delay showing the tooltip.
    */
-  show(position: TooltipPosition, delay: number): void {
+  show(delay: number): void {
```

Comment:
This is a breaking API change; causing some issues trying to sync

Reply:
I can see how it could break somebody, but this is on a private component that is only exposed through an underscored property.

Reply:
It's from a team that's extending the tooltip. I guess I can update them.